### PR TITLE
Adding another example

### PR DESCRIPTION
**Pull request by marcemiraval**
Add data and rmd file with another leaflet example

### CODE REVIEW

File: hospitals.Rmd [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo=FALSE, warning=FALSE)
+```
+
+```{r load_libraries}
+library(dplyr)
+library(leaflet)
+library(sf)
+library(htmlwidgets)
+library(rgdal)
```
Comment (mikejohnson51) [inline hospitals.Rmd]: is rgdal needed, I don't see it being used? its a big package if not :)

File: hospitals.Rmd [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+```
+
+```{r read_data}
+hospitals_bogota <- readRDS(file = "hospitals_bogota.rds")
+```
+
+```{r data_wrangling}
+hospitals_bogota <- hospitals_bogota %>% 
+  select(-c(CITY, pais, SurgRms, IntensRms, EmergRms, CTRYISOA3, ADM1ISOC, ADM1ISON, SubRegion, PRIORITY, Map)) %>% 
+  mutate(lat = as.numeric(Latitude), long = as.numeric(Longitude)) %>%
```
Comment (mikejohnson51) [inline hospitals.Rmd:25]: override to reduce select? eg. Latitude = as.numeric(Latitude))?

File: hospitals.Rmd [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+                             "Big - High medical specialties"), 
+                  sizes = c(8, 15, 25), # circles size in legend look diferent so I had to exxagerage (increase) values
+                  stroke = FALSE) %>% 
+  addLegend(pal = pal, values = ~H_Sector, title = "Hospital Sector")
+}
+```
+
+
+```{r mapping_hazard}
+seismic_hazard <- st_read("respuesta_sismica/Respuesta_Sismica.shp")
```
Comment (mikejohnson51) [inline hospitals.Rmd]: as st_transform here as pipe?

File: hospitals.Rmd (hunk 1 above)
Comment (marcemiraval) [reply hospitals.Rmd]: Line deleted! It wasn't necessary :)

File: hospitals.Rmd (hunk 2 above)
Comment (marcemiraval) [reply hospitals.Rmd:25]: You are right! I tried to simplify code here

File: hospitals.Rmd (hunk 3 above)
Comment (marcemiraval) [reply hospitals.Rmd]: Will import polygon data later

File: hospitals.Rmd [hunk 4]
```diff
@@ -44,20 +62,26 @@ addLegendCustom <- function(map, colors, labels, sizes, opacity = 0.8, stroke, t
...
+                   label = ~Hospital_name, 
                    popup = paste0("Surgery Rooms:", hospitals_bogota$SurgeryRm)) %>%
   addLegendCustom(colors = c("gray", "gray", "gray"), 
                   labels = c("Small - General surgery and open 24 hrs", 
                              "Medium -Two Specialty services", 
                              "Big - High medical specialties"), 
                   sizes = c(8, 15, 25), # circles size in legend look diferent so I had to exxagerage (increase) values
-                  stroke = FALSE) %>% 
+                  stroke = FALSE) %>%
+# addLegend(pal = pal_hazard, values = ~reclass, title = "Seismic Hazard") %>% # Does not work
```
Comment (marcemiraval) [inline hospitals.Rmd:84]: Still need to figure out what is the problem with the legend for the hazards information. This line.

**Pull request merged**